app/uk/gov/hmrc/thirdpartyapplication/domain/models/Event.scala

```diff
@@ -80,8 +91,25 @@ object UpdateApplicationEvent {
     eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
     actor: Actor,
     oldLocation: PrivacyPolicyLocation,
-    newLocation: PrivacyPolicyLocation
-  ) extends UpdateApplicationEvent
+    newLocation: PrivacyPolicyLocation,
+    requestingAdminEmail: String
+  ) extends UpdateApplicationEvent with TriggersStandardChangedNotification {
```

Comment:
Can we not pollute the event with this?  We could put this extract logic into the notification service where it probably belongs.

Reply:
I wanted to make it consistent with the existing ProductionAppNameChanged event, which has this field already.